packages/notifier-seeder/package.json

```diff
@@ -28,13 +28,12 @@
     "prettier": "2.8.8",
     "ts-node": "10.9.2",
     "typescript": "5.1.3",
-    "dotenv-flow": "3.2.0",
```

Comment:
Should this be removed? It was not removed in other package.json files

Reply:
This was in both `dependencies` and `devDependencies`, I removed it from the `devDependencies`